api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -2296,8 +2296,113 @@ message ExportUpgradeWindowsResponse {
   string SystemdUnitSchedule = 3;
 }
 
+// AssistantMessageRequest is a request to the assistant service.
+message AssistantMessageRequest {
+  // conversationId it's the ID of a conversation.
+  // It's used to tie all messages in a one conversation.
+  string conversation_id = 1;
+
+  // username is a username of the user who sent the message.
+  string username = 2;
+}
+
+// AssistantMessage is a message sent to the assistant service. The conversation
+// must be created first.
+message AssistantMessage {
+  // conversationId used to tie all messages in a one conversation.
+  string conversation_id = 1;
+
+  // username is a username of the user who sent the message.
+  string username = 2;
+
+  // type is a type of message. It can be Chat response/query or a command to run.
+  string type = 3;
```

Comment:
Would an enum make more sense?

Reply:
Yes and no. From the protobuf perspective, yes, but WebUI is also using this field, and to not pass super long strings to UI or keep re-mapping between values, I thought that `string` is just simpler.